compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Version.hs

```diff
@@ -51,6 +51,12 @@ data Feature = Feature
 -- featureTextCodePoints :: Feature
 -- featureTextCodePoints = Feature "Conversion between text and code points" version1_6
 
+featureNumeric :: Feature
+featureNumeric = Feature
+    { featureName = "Numeric type"
+    , featureMinVersion = versionDev -- TODO: Update when stabilized.
```

Comment:
iirc there is a tracking issue for this somewhere right? Maybe link to that?

Reply:
I would add `version1_7 = versionVDev` above, add it to `supportedOutputVersions`, and use `version1_7` here. Then rolling out version 1.7 just requires switching to `version1_7 = V1 $ PointStable 7`.

Reply:
Ah, yes! Thanks!